boards/x86/qemu_x86/doc/index.rst

```diff
@@ -8,13 +8,6 @@ Overview
 
 The X86 QEMU board configuration is used to emulate the X86 architecture.
 
-.. figure:: qemu_x86.png
-   :width: 600px
-   :align: center
-   :alt: Qemu
-
-   Qemu (Credit: qemu.org)
```

Comment:
Hi @gmarull , just double check it. The qemu logo photo will be removed, right?

Reply:
yes, I removed all instances of it (other logos as well)